### PR TITLE
VIT-3432 MFP documentation

### PR DESCRIPTION
**Pull request by ib250**
## Description

Update MFP Guide

### CODE REVIEW

**Review by maitham — APPROVED** [review]

File: docs/wearables/providers/guides/my_fitness_pal.mdx [hunk 1]
```diff
@@ -4,7 +4,23 @@ title: "MyFitnessPal (Beta)"
 
 ## Connecting MyFitnessPal to Vital
 
-You can now link MyFitnessPal accounts to Vital! You can do this with Vital Link Widget or programatically like other [password](/api-reference/link/link-email-provider) providers.
+You can now link MyFitnessPal accounts to Vital! You can do this with Vital Link Widget or programatically like other [password](/api-reference/link/link-password-provider) providers.
```
Comment (maitham) [inline docs/wearables/providers/guides/my_fitness_pal.mdx:7]: Should be separate words

File: docs/wearables/providers/guides/my_fitness_pal.mdx [hunk 2]
```diff
@@ -4,7 +4,23 @@ title: "MyFitnessPal (Beta)"
...
+
+## Before Connecting
+
+You can share your MyFitnessPal diary with Vital by locking your diary with key.
```
Comment (maitham) [inline docs/wearables/providers/guides/my_fitness_pal.mdx:11]: Can we describe this a bit better

File: docs/wearables/providers/guides/my_fitness_pal.mdx [hunk 3]
```diff
@@ -4,7 +4,23 @@ title: "MyFitnessPal (Beta)"
...
+- Go to [your diary settings](https://www.myfitnesspal.com/account/diary-settings) and click **locked with key**.
+
```
Comment (maitham) [inline docs/wearables/providers/guides/my_fitness_pal.mdx:13]: Maybe include the mobile instructions too?

File: docs/wearables/providers/guides/my_fitness_pal.mdx (hunk 1 above)
Comment (ib250) [reply docs/wearables/providers/guides/my_fitness_pal.mdx:7]: Which words?

File: docs/wearables/providers/guides/my_fitness_pal.mdx (hunk 2 above)
Comment (ib250) [reply docs/wearables/providers/guides/my_fitness_pal.mdx:11]: You resolved this, are you sure you are happy with the wording?

File: docs/wearables/providers/guides/my_fitness_pal.mdx (hunk 1 above)
Comment (RuiAAPeres) [reply docs/wearables/providers/guides/my_fitness_pal.mdx:7]: To be fair MyFitnessPal is all together:

image
image

**Comment by RuiAAPeres** [comment]
The screenshot needs to be fixed, alongside the widget instructions. MyFitnessPal is all together.

File: docs/wearables/providers/guides/my_fitness_pal.mdx (hunk 3 above)
Comment (ib250) [reply docs/wearables/providers/guides/my_fitness_pal.mdx:13]: I've added mobile instructions

**Comment by ib250** [comment]
> The screenshot needs to be fixed, alongside the widget instructions. MyFitnessPal is all together.

What do I change?

**Comment by RuiAAPeres** [comment]
This bit where it should be MyFitnessPal. 

image

**Comment by ib250** [comment]
> This bit where it should be MyFitnessPal.
> 
> image

Ok, I'll update that

File: docs/wearables/providers/guides/my_fitness_pal.mdx (hunk 3 above)
Comment (ib250) [reply docs/wearables/providers/guides/my_fitness_pal.mdx:32]: @maitham please check out the mobile instructions when you get a chance

**Pull request merged**